client/components/tinymce/plugins/simple-payments/plugin.jsx

```diff
@@ -9,16 +9,15 @@ import { unmountComponentAtNode } from 'react-dom';
  * Internal Dependencies
  */
 import SimplePaymentsDialog from './dialog';
+import { serialize } from './shortcode-utils';
 import { renderWithReduxStore } from 'lib/react-helpers';
 
 const simplePayments = editor => {
 	let node;
 	const store = editor.getParam( 'redux_store' );
 
 	editor.on( 'init', () => {
-		node = editor.getContainer().appendChild(
-			document.createElement( 'div' )
-		);
+		node = editor.getContainer().appendChild( document.createElement( 'div' ) );
```

Comment:
> Would it help if I moved the formatting-only changes to a separate commit?

Hmm, not sure. Is there a Calypso-wide consensus of using Prettier for code formatting? If yes, we should all use it so the codebase is always looking similar. If it's not, there will be always a problem that a person who uses Prettier will unnecessarily modify a code. Alternative: couldn't you configure Prettier to only format the code you introduced in a PR? That would make complete sense to me.